packages/pulpcore/python-pulpcore/python-pulpcore.spec

```
@@ -13,61 +13,59 @@ BuildArch:      noarch
 
 BuildRequires:  python3-devel
 BuildRequires:  python3-setuptools
-BuildRequires:  python3-setuptools < 42.1.0
-BuildRequires:  python3-setuptools >= 39.2.0
 
 %description
-Pulp is a platform for managing repositories of content, such as software
-packages, and pushing that content out to large numbers of consumers.
+Pulp is a platform for managing repositories of content, such as software packages, and pushing that content out to large numbers of consumers.	
 
-Using Pulp you can:
-- Locally mirror all or part of a repository
-- Host your own content in a new repository
-- Manage content from multiple sources in one place
+Using Pulp you can:	
+- Locally mirror all or part of a repository	
+- Host your own content in a new repository	
+- Manage content from multiple sources in one place	
 - Promote content through different repos in an organized way
 
 %package -n     python3-%{pypi_name}
 Summary:        %{summary}
 %{?python_provide:%python_provide python3-%{pypi_name}}
+Requires:       python3-Django >= 2.2.3
+Conflicts:      python3-Django >= 2.3
+Requires:       python3-PyYAML < 5.4.0
+Requires:       python3-PyYAML >= 5.1.1
```

```suggestion
Requires:       python3-django >= 2.2.3
Conflicts:      python3-django >= 2.3
Requires:       python3-pyyaml < 5.4.0
Requires:       python3-pyyaml >= 5.1.1
```

we use downcase package names